test/over_react/component_declaration/flux_component_test/prop_validation.dart

```diff
@@ -0,0 +1,24 @@
+part of over_react.component_declaration.flux_component_test;
+
+@Factory()
+UiFactory<TestPropValidationProps> TestPropValidation;
+
+@Props()
+class TestPropValidationProps extends FluxUiProps {
+  @requiredProp
+  String required;
+}
+
+@Component()
+class TestPropValidationComponent extends FluxUiComponent<TestPropValidationProps> {
+  int numberOfRedraws = 0;
+
+  @override
+  render() => Dom.div()();
+
+  @override
+  void setState(_, [callback()]) {
```

Comment:
What is this for?

Reply:
Looks like it was unnecessary, copypasta'd from one of the other components.

Removed in https://github.com/Workiva/over_react/pull/193/commits/20bbd9e83a8f6bdd7ee04fdba711a3bc4d4dc710